.github/workflows/trivy-scan.yml

```diff
@@ -46,7 +46,6 @@ jobs:
           exit 1
         fi
     - name: Run trivy scan
```

Comment:
question: did you run `go mod tidy` in the end?

Reply:
Yes, I did.